faraday/proxy.py

```diff
@@ -452,13 +463,13 @@ def proxy():
         # present, create port queue for it and append data to that queue
         try:
```

Comment:
Instead of a try/catch you can use `if "data" in data:` or `if "data" not in data:`.